src/pages/interim/faq/_love-trading.tsx

```diff
@@ -4,14 +4,16 @@ import { Localize } from 'components/localization'
 
 const content = {
     left: {
-        header: <Localize translate_text="Be among the first to try Deriv.com" />,
-        button_text: <Localize translate_text="Visit Deriv.com now" />,
+        header: <Localize translate_text="_t_Be among the first to try Deriv.com_t_" />,
+        button_text: <Localize translate_text="_t_Visit Deriv.com now_t_" />,
         button_url: '/',
     },
     image: 'smart_trader',
     right: {
-        header: <Localize translate_text="Love trading on Binary.com’s signature platform?" />,
-        button_text: <Localize translate_text="Try SmartTrader on Deriv.com" />,
+        header: (
+            <Localize translate_text="_t_Love trading on Binary.com’s signature platform?_t_" />
```

Comment:
same

Reply:
@niloofar-deriv  I have skipped all the above since it is not part of an array and feel like it wont make any difference when move it to the final element. what do you think?

Reply:
IMO it's better to use `TString` format and don't pass `Localize` everywhere, It's much cleaner

Reply:
Ok , then we go like that. Thank you @niloofar-deriv 👍🏾